btrack/utils.py

```diff
@@ -243,6 +244,9 @@ def update_segmentation(
         ordered T(Z)YX. Assumes that this is not binary and each object has a unique ID.
     tracks : list[btypes.Tracklet]
         A list of :py:class:`btrack.btypes.Tracklet` objects from BayesianTracker.
+    scale : tuple, optional
```

Comment:
Can you specify the axis order here? Is it xyz or zyx?

Reply:
xyz

Reply:
z needs to be last for accessing the indexes of x and y if z isnt present